src/main/java/ua/softserveinc/tc/constants/EntryConstants.java

```diff
@@ -6,7 +6,7 @@
 
     public static final String ENTRY_POINT = "entrypoint";
 
-    public static final String ALLOWED_CUSTOM_AUTHENTICATION = "login.without.sso";
+    public static final String ALLOWED_CUSTOM_AUTHENTICATION = "enable.custome.login";
 
```

Comment:
Maybe you want to write custom? I think it  is very bewildered name. I think it is the  best to call it as -> multiple.choose.login. Or something like that.